handlers.js

```diff
@@ -301,7 +301,7 @@ module.exports = function(port) {
 
         Promise.resolve()
         .then(() => {
-            if (port.config.identityNamespace === false) {
+            if (port.config.identityNamespace === false || (request.payload.method !== identityCheckFullName && !request.auth.strategy)) {
```

Comment:
maybe it will be better to have some parameter that will tell explicitly that we need to skip identity.check